src/app/Login.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState } from 'react';
+
+import styles from './Login.module.css';
+
+interface LoginProps {
+  goLogin: () => void;
+}
+
+const Login: React.FC<LoginProps> = ({ goLogin }) => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [token, setToken] = useState<string>('');
+  const [nickname, setNickname] = useState('');
```

Comment:
빈 값게 `''` 보다는 null을 사용하면 더 명확하고 타입스크립트를 잘 활용할 수 있을 것 같아요 (이거 반영하면 아래쪽 `!== ''` 들도 `!== null` 로 수정되어야 하겠습니다)

```suggestion
  const [token, setToken] = useState<string | null>(null);
  const [nickname, setNickname] = useState<string | null>(null);
```

Reply:
확인했습니다!